File: [1] BEGINNER/1071 - Soma de Impares Consecutivos I.py
'''
Leia 2 valores inteiros X e Y. A seguir, calcule e mostre a soma dos números impares entre eles.

Entrada
O arquivo de entrada contém dois valores inteiros.

Saída
O programa deve imprimir um valor inteiro. Este valor é a soma dos valores ímpares que estão entre os valores fornecidos na entrada que deverá caber em um inteiro.
'''

a = int(input())
b = int(input())
X = min(a,b)
Y = max(a,b)
impares = []
for i in range(X+1, Y):
    if i % 2 != 0:
        impares.append(i)
print(sum(impares))

File: [1] BEGINNER/1071 - Soma de Impares Consecutivos I.rb
=begin
Leia 2 valores inteiros X e Y. A seguir, calcule e mostre a soma dos números impares entre eles.

Entrada
O arquivo de entrada contém dois valores inteiros.

Saída
O programa deve imprimir um valor inteiro. Este valor é a soma dos valores ímpares que estão entre os valores fornecidos na entrada que deverá caber em um inteiro.
=end

x = gets.chomp.to_i
y = gets.chomp.to_i

valores = [x,y]
count = 0

for i in (valores.min..valores.max)
    if i != valores.min and i != valores.max and i % 2 != 0
        count += i 
    end
end

puts count